regenerate.sh

```diff
@@ -45,19 +45,6 @@ mkdir -p ${WORKDIR}/googleapis/google/rpc
 echo "curl https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto"
 curl --silent https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto > ${WORKDIR}/googleapis/google/rpc/code.proto
 
-# Pull in the following repos to build the MeshCA config proto.
-ENVOY_API_REPOS=(
-  "https://github.com/envoyproxy/data-plane-api"
-  "https://github.com/cncf/udpa"
-  "https://github.com/envoyproxy/protoc-gen-validate"
-)
-for repo in ${ENVOY_API_REPOS[@]}; do
-  dirname=$(basename ${repo})
-  mkdir -p ${WORKDIR}/${dirname}
-  echo "git clone ${repo}"
-  git clone --quiet ${repo} ${WORKDIR}/${dirname}
-done
-
 # Pull in the MeshCA service proto.
```

Comment:
What will it take to do this next?  We should really not be generating `.pb.go`s from protos outside `grpc-proto` at all.

Reply:
Yes, this is a weird case. Ideally, the MeshCA protos should have been part of googleapis or some public repo like that.

When (and if) we have to get this back, we probably might directly depend on the `pb.go`s from the istio repo. But I'm not sure if that is any better.

Reply:
> When (and if) we have to get this back, we probably might directly depend on the pb.gos from the istio repo. But I'm not sure if that is any better.

It is better, since it means:

1. we don't have to deal with their proto's dependencies, which may expand over time, and
2. we don't risk a panic if multiple generated versions of this proto are imported by one program.

We should be actively looking to remove this.

Reply:
Filed an issue to track this: https://github.com/grpc/grpc-go/issues/4058